modules/auxiliary/scanner/motorola/timbuktu_udp.rb

```diff
@@ -52,8 +52,11 @@ def run_host(ip)
         else
           print_error("Unable to determine info for #{ip}...")
         end
+    rescue ::Errno::EPIPE, ::Rex::HostUnreachable, ::Rex::ConnectionTimeout, ::Rex::ConnectionRefused => e
+      vprint_error(e.message)
+      elog("#{e.class} #{e.message}\n#{e.backtrace * "\n"}")
```

Comment:
If all you're doing is printing and logging the error, you shouldn't bother `rescue`ing it in the first place. Just let the console handle it.